lib/auth/resetpasswordtoken.go

```diff
@@ -126,6 +127,14 @@ func (s *AuthServer) CreateResetPasswordToken(ctx context.Context, req CreateRes
 		return nil, trace.Wrap(err)
 	}
 
+	if err := s.EmitAuditEvent(events.ResetPasswordTokenCreated, events.EventFields{
+		events.FieldName:             req.Name,
```

Comment:
Lets keep the scope of stratus as small as possible to make it's easy to merge with master. If this change is not required, lets undo (i assume you are following @russjones suggestion to move audit events from the _with_roles.go layer)